.gitignore

```diff
@@ -0,0 +1,3 @@
+/.idea
```

Comment:
Je .idea map is wel op github terecht gekomen, heb je dit mischien pas toegevoegd na dat je je main gepushed hebt?